cpp/include/cudf/strings/translate.hpp

```diff
@@ -50,6 +53,48 @@ std::unique_ptr<column> translate(
   std::vector<std::pair<char_utf8, char_utf8>> const& chars_table,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Removes or keeps the specified character ranges in cudf::strings::filter_characters
+ */
+enum class filter_type : bool { KEEP, REMOVE };
+
+/**
+ * @brief Removes ranges of characters from each string in a strings column.
+ *
+ * This can also be used to keep only the specified character ranges
+ * and remove all others from each string.
+ *
+ * @code{.pseudo}
+ * Example:
+ * s = ["aeiou","AEIOU","0123456789"]
+ * f = [{'a','l'}, {'M','Z'}, {'4','6'}]
```

Comment:
nitpick: the example makes it look like the vector of filter char pairs needs to be the same size as the strings column and that the filters are for corresponding elements in the column.

Reply:
That is a good point since each range coincidentally only applies to the corresponding element. I should improve the example.